src/main/java/frc/robot/Robot.java

```diff
@@ -249,21 +230,20 @@ public void periodic() {
 		CommandScheduler.getInstance().run(); // Should be last
 	}
 
-	public Command getAuto() {
-		return whereToScoreFirstObjectChooser.get()
-			.get()
-			.andThen(
-				PathPlannerAutoWrapper
-					.chainAutos(
-						whereToIntakeSecondObjectChooser.getChosenValue(),
-						whereToScoreSecondObjectChooser.getChosenValue(),
-						whereToIntakeThirdObjectChooser.getChosenValue(),
-						whereToScoreThirdObjectChooser.getChosenValue(),
-						whereToIntakeFourthObjectChooser.getChosenValue(),
-						whereToScoreFourthObjectChooser.getChosenValue()
-					)
-					.asProxy()
-			);
+	public PathPlannerAutoWrapper getAuto() {
+		return PathPlannerAutoWrapper.chainAutos(
+			whereToScoreFirstObjectChooser.getChosenValue(),
+			PathPlannerAutoWrapper
+				.chainAutos(
+					whereToIntakeSecondObjectChooser.getChosenValue(),
```

Comment:
```suggestion
					secondObjectIntakePlaceChooser.getChosenValue(),
```

Might be better, if you agree then change for all of them